assets/js/base/components/chip/index.js

```diff
@@ -25,46 +25,79 @@ const Chip = ( {
 	onRemove = () => {},
 	disabled = false,
 	radius = 'small',
+	removeOnClick = false,
```

Comment:
> So with that in mind, the prop name seems like it could use some "tightening up" so it's purpose is a bit more intuitive. Maybe something like `removeOnAnyClick`?

:+1:  Done in ce51de2117cceec4305817932a33caf36045917a.

> Also, I wonder, will there ever be any use-case for using this same Chip Pattern for something that is _not_ removable (meaning the chip is persistently displayed)? Should we differentiate the name of this particular component so it's clear it's always removable (i.e. `RemovableChip` or `DismissableChip`)? Eventually the ui/ux could be moved into `<Chip>` and this component (<DismissibleChip>) would wrap around the former enhancing it with the removal actions/labels.

Yes, that sounds good to me. Just thinking we should wait until we see the real need to have an un-removable chip before differentiating between two kinds of chips?

Reply:
> Just thinking we should wait until we see the real need to have an un-removable chip before differentiating between two kinds of chips?

I'd be okay with that, but I just wonder if it would be better to be more explicit _now_ with the name of the component and call it `RemovableChip` or `DismissibleChip` which leaves room for having just `Chip` down the road. It makes it a bit clearer that this is a component with dismissing behaviour?

Reply:
I ended up splitting the component in two (`Chip` and `RemovableChip`) as you suggested in the first place. This way both components can share the same CSS classes so it's not a breaking change: d0eeb266a71d498fdcdff3f4826185335e19da93.